src/python-porting-guide.md

```diff
@@ -0,0 +1,18 @@
+Guide for porting activities from Python 2 to Python 3
+=======================================
+
+**Note:** Kindly use Python 3.6.3
```

Comment:
Let's not say specific version required, but instead say that these instructions were tested with 3.6.3.  And move it to end of file, as it breaks flow.